extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -291,6 +295,11 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     }
     this.ampAnalyticsConfig_ = null;
   }
+
+  /** @override */
+  getPreconnectUrls() {
+    return 'https://googleads.g.doubleclick.net';
```

Comment:
to be specific, so i want the method to return `{!Array<string>}`. So the caller can to `results.length` freely